viewmodels/src/androidMain/kotlin/com/mirego/trikot/viewmodels/PickerViewModelBinder.kt

```diff
@@ -0,0 +1,51 @@
+package com.mirego.trikot.viewmodels
+
+import android.view.View
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import android.widget.Spinner
+import androidx.databinding.BindingAdapter
+import com.mirego.trikot.streams.reactive.asLiveData
+import com.mirego.trikot.streams.reactive.observe
+
+object PickerViewModelBinder {
+    @Suppress("UNCHECKED_CAST")
+    @JvmStatic
+    @BindingAdapter("view_model", "lifecycleOwnerWrapper")
+    fun bind(
+        picker: Spinner,
+        pickerViewModel: PickerViewModel<*>?,
+        lifecycleOwnerWrapper: LifecycleOwnerWrapper
+    ) {
+        pickerViewModel?.let { viewModel ->
+            viewModel.elements.observe(lifecycleOwnerWrapper.lifecycleOwner) { list ->
+                picker.adapter =
+                    ArrayAdapter(picker.context, android.R.layout.simple_spinner_item, list.map { it.displayName })
```

Comment:
Can't we reuse the same `adapter` and simply submit a new item list when `elements` change?